script/QA/QA_15.txt

```diff
@@ -154,6 +154,7 @@ Then...?
 -- Page 71, Offset 43448.
 -- 今回の遠野くんの状態より酷い目に
 // Uhhh game seems to break for me here?
+// Game also breaks for me. My allscr is super old though
```

Comment:
Yeah, I made some allscr changes for this and one of the other QAs to work around a hang. Should be sorted if you rebuild.